recipe/meta.yaml

```
@@ -23,28 +22,31 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - llvm-openmp    # [osx]
   host:
     # Host dependencies are pinned to last build
     - python
     - cython 0.29.33
-    - llvm-openmp 14.0.6  # [osx]
     - numpy   1.19   # [py<310]
     - numpy   1.21   # [py==310]
     - numpy   1.23   # [py>=311]
+    - llvm-openmp 14.0.6  # [osx]
     - pip
     - scipy 1.9.3   # [py<311]
     - scipy 1.10.0  # [py>=311]
```

```suggestion
    - scipy 1.10.0  # [py>=311]
    - scipy 1.11.1  # [py==312]
```
(for my futur self :D )